tests/test_meshtastic_utils.py

```
@@ -17,9 +17,10 @@
 import sys
 import threading
 import unittest
+from collections.abc import Generator
 from concurrent.futures import TimeoutError as ConcurrentTimeoutError
 from types import SimpleNamespace
-from typing import Any, Callable
+from typing import Any, Callable, NoReturn
```

Ruff (0.15.10)

[warning] Import from `collections.abc` instead: `Callable`

Import from `collections.abc`

(UP035)